config.lua

```diff
@@ -1 +1,4 @@
 Config = Config or {}
+
+Config.Inv = "ox" -- qb(=lj) or ox [Inventory system]
+Config.webhook = "https://discord.com/api/webhooks/1098347143602913340/61nQVpgAZILNJyUNN3v6AP2yvtR8VLKcOjNVcfGYpKBfotrpqNy4uDxsQA-ERh4bqnFi" -- Add Discord webhook
```

Comment:
Move this to server sided please, so people can't dump it
And leave it as default qb, and remove your webhook

Thanks!